src/magnify/apps/core/models.py

```diff
@@ -516,7 +525,7 @@ def __str__(self):
     @property
     def jitsi_name(self):
         """The name used as Jitsi room for this meeting."""
```

Comment:
I think this method naming and documentation are outdated. Also, tests still mentioned `jitsi`